src/components/AsideSection/AsideSection.tsx

```diff
@@ -27,11 +27,11 @@ export default class AsideSection extends React.Component<AsideSectionProps, {}>
   render() {
     return (
       <div className="y-aside-section">
-        <Block textSize={TextSize.SMALL}>
+        <Block>
```

Comment:
you can give Block `push={2}` instead of adding the 2px of CSS

Reply:
Well, it won't really do what we want, we need spacing between header and border. Applying push={2} to the above Block doesn't do the thing